tools/mapmerge2/map_scripts/clothingunderrepath.txt

```diff
@@ -119,7 +119,7 @@
 /obj/item/clothing/under/lawyer/blacksuit : /obj/item/clothing/under/suit/black
 /obj/item/clothing/under/lawyer/blacksuit/skirt : /obj/item/clothing/under/suit/black/skirt
 /obj/item/clothing/under/lawyer/really_black : /obj/item/clothing/under/suit/black_really
-/obj/item/clothing/under/lawyer/really_black/skirt : /obj/item/clothing/under/suit/black_really/skirt
+/obj/item/clothing/under/lawyer/black/female : /obj/item/clothing/under/suit/black/female
```

Comment:
Shouldn't this be the old one on the left and the new one on the right?

Reply:
No fucking clue.

Reply:
No this is correct

Reply:
Alrighty then, all good then